integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -444,6 +450,172 @@ public void testLivenessProbe() {
+    assertFalse(assertDoesNotThrow(
+        () -> doesFileExistInPod(domainNamespace, adminServerPodName, dataFileToCheck),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("%s exists in pod %s in namespace %s, expects not exist",
+            dataFileToCheck, adminServerPodName, domainNamespace));
+
+    // check in admin server pod, the default admin server data file moved to DATA_HOME_OVERRIDE
+    String defaultAdminDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/_WLS_ADMIN-SERVER000000.DAT";
+    assertTrue(assertDoesNotThrow(() ->
+        doesFileExistInPod(domainNamespace, adminServerPodName, defaultAdminDataFile),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("can not find file %s in pod %s in namespace %s",
+            defaultAdminDataFile, adminServerPodName, domainNamespace));
+
+    // check in managed server pod, the custom data file for JMS and default managed server datafile are created
+    // in DATA_HOME_OVERRIDE
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = domainUid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      String customDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/FILESTORE-0@MANAGED-SERVER" + i + "000000.DAT";
+      assertTrue(assertDoesNotThrow(() ->
+              doesFileExistInPod(domainNamespace, managedServerPodName, customDataFile),
+          String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+              customDataFile, managedServerPodName, domainNamespace)),
+          String.format("can not find file %s in pod %s in namespace %s",
+              customDataFile, managedServerPodName, domainNamespace));
+
+      String defaultMSDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/_WLS_MANAGED-SERVER" + i + "000000.DAT";
+      assertTrue(assertDoesNotThrow(() ->
+              doesFileExistInPod(domainNamespace, managedServerPodName, defaultMSDataFile),
+          String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+              defaultMSDataFile, managedServerPodName, domainNamespace)),
+          String.format("can not find file %s in pod %s in namespace %s",
+              defaultMSDataFile, managedServerPodName, domainNamespace));
+    }
+  }
+
+  @Test
+  @DisplayName("Test dataHome override in a domain with model in image type")
+  public void testDataHomeOverrideMiiDomain() {
+    // in this domain, dataHome is not specified
+    // the domain contains JMS and File Store configuration
+    // File store directory is set to /u01/oracle/customFileStore/ in the model file which should not be overridden
+    // by dataHome
+    // File store and JMS server are target to admin server
```

Comment:
File store and JMS server are targeted to the admin server

Reply:
fixed